libraries/RC_Channel/RC_Channel.h

```diff
@@ -152,14 +152,12 @@ class RC_Channels {
 
     static uint8_t get_valid_channel_count(void);                      // returns the number of valid channels in the last read
     static int16_t get_receiver_rssi(void);                            // returns [0, 255] for receiver RSSI (0 is no link) if present, otherwise -1
-    static bool has_new_input(void);                                   // returns true if there has been new input since last checked
+    static bool read_input(void);                                      // returns true if there has been new input has been read in
```

Comment:
Is your English getting worse? 😄 Change to something like _returns true if new input has been read in_

Reply:
Unavoidable conclusion: Yes.